dandi/cli/command.py

```diff
@@ -175,6 +175,15 @@ def main(log_level, pdb=False):
         from ..utils import setup_exceptionhook
 
         setup_exceptionhook()
+    try:
+        import etelemetry
```

Comment:
mental note: apparently according to coverage annotation - none of this code is tested, so whatever invocation we use in the tests, it is not using this top level `main`